packages/clay-card/src/CardWithUser.tsx

```diff
@@ -19,12 +19,9 @@ interface IProps {
 	actions?: React.ComponentProps<typeof ClayDropDownWithItems>['items'];
 
 	/**
-	 * Labels for the aria attributes
+	 * Props to add to the dropdown trigger element
 	 */
-	ariaLabels?: {
-		cardCheckbox: string;
-		dropDownTrigger: string;
-	};
+	checkboxProps?: React.HTMLAttributes<HTMLInputElement>;
```

Comment:
Same as above. This is even more important for form elements since they should be able to also add a `name` attr if needed.